src/templates/blog.tsx

```diff
@@ -0,0 +1,79 @@
+import { graphql } from 'gatsby';
+import React from 'react';
+import Article from '../components/Article';
+import Layout from '../components/Layout';
+import { LearnPageContext, LearnPageData } from '../types';
+
+import '../styles/article-reader.scss';
+import '../styles/learn.scss';
+import Footer from '../components/Footer';
+
+interface Props {
+  data: LearnPageData;
+  pageContext: LearnPageContext;
+}
+const LearnLayout = ({
+  data,
+  pageContext: { next, previous, relativePath },
+}: Props): React.ReactNode => {
+  const {
+    doc: {
+      frontmatter: { title, description },
+      html,
+      tableOfContents,
+      fields: { authors },
+    },
+  } = data;
+  return (
+    <>
+      <Layout title={title} description={description} showFooter={false}>
+        {/* <main className="grid-container"> */}
```

Comment:
remove any commented out code

Reply:
The PR was not finished yet actually. I Will do that after adding tests, some more features and fixing some responsiveness issues.

Reply:
sounds good 👍🏻